### ISSUE TITLE
Add lxhentai change domain option feature like hentavn

### ISSUE THREAD

**Issue by leonminus** [keiyoushi/tachiyomi-extensions-source#2715]
### Source name

Lxhentai

### Source language

Lxhentai

### Describe your suggested feature

Change url domain in-app option like hentaivn does because in our country these "unhealthy" site are usually get blocked by the nation net provider (dunno how to call it) so they have to change domain many times

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/mihonapp/mihon/issues/new/choose).
- [X] I have updated the app to version **[0.15.3](https://github.com/mihonapp/mihon/releases/latest)**.
- [X] I will fill out all of the requested information in this form.
